doc/tutorials/mount.md

```diff
@@ -110,22 +127,60 @@ Let us mount a projects git configuration into the dir namespace:
     # mount gits configuration into Elektra
     sudo kdb mount /.git/config dir/git ini multiline=0
 
-As git uses the ini format for its configuration we use the ini plugin.
+As git uses the ini format for its configuration we use the [ini plugin](src/plugins/ini/README.md).
 You can pass parameters to plugins during the mount process. This is what
 we did with `multiline=0`. Git intends the entries in its configuration
 files and the default behaviour of the ini plugin is to interpret these indented
-entries as values that span mutiple lines. With the passed parameter disable
-this behaviour and make the ini-plugin compatible to git.
+entries as values that span mutiple lines. The passed parameter disables
+this behaviour and makes the ini-plugin compatible with git configuration.
+
+Now let us see how smoothly the ini plugin sets and gets the git configuration.
+
+    # set a user name ...
+    git config user.name "Rob Banks"
+
+    # ... and read it with kdb
+    kdb get dir/git/user/name
+    Rob Banks
+
+    # set a user email with kdb ...
+    kdb set dir/git/user/email "rob.banks@dot.com"
 
-Alternatively we could have mounted another file format:
+    # and read it with git
+    git config --get user.email
+    rob.banks@dot.com
+
+
+#### Meta data ####
+
+// TODO: work on this subchapter:
+[] find nice example
+[] link to validation.md
+
+Elektra is able to store [meta data](doc/help/elektra-metadata.md) of keys, provided the format of the file that holds the configuration supports this feature.
+The ini plugin doesn't support this feature, but the [ni](src/plugins/ni/README.md) and the [dump](src/plugins/dump/README.md) plugin do.
```

Comment:
The sentence is correct, but we should explain here additionally that the ini plugin creates some meta data on its own to remember comments and ordering of keys within the files. (Try to export an ini file via dump/ni; hosts does similar things).